src/kernel/kmain.c

```diff
@@ -58,7 +58,9 @@ void check_interrupts() {
 
     uint32_t tick = timer_tick();
 
-    while (tick == timer_tick()) ;
+    while (tick == timer_tick()) {
+        ;
```

Comment:
Ditto

Reply:
Yep, I don't like it much either and I am still experimenting. If things like this get automated and it is consistent everywhere, I don't mind though.

Reply:
Edit: I reverted those changes in #44